core/src/avm2/namespace.rs

```diff
@@ -153,10 +155,8 @@ impl<'gc> Namespace<'gc> {
             let stripped = strip_version_mark(namespace_name.as_wstr());
             let has_version_mark = stripped.is_some();
             if let Some((stripped, version)) = stripped {
-                assert!(
-                    is_playerglobals,
-                    "Found versioned url {namespace_name:?} in non-playerglobals domain"
-                );
+                // Ignore if versioned url is in non-playerglobals domain as avmplus does
+                // https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/AbcParser.cpp#L1510
                 let stripped_string = AvmString::new(context.gc_context, stripped);
                 namespace_name = context.interner.intern(context.gc_context, stripped_string);
                 api_version = version;
```

Comment:
This will set the api version to the value parsed from the mark, which isn't correct.